requirements.txt

```diff
@@ -19,9 +19,8 @@ djangorestframework==3.13.1
 djangorestframework-camel-case==1.3.0
 docutils==0.18.1
 drf-yasg[validation]==1.20.0
-ethereum==2.3.2
```

Comment:
Why was this one removed?

Reply:
It was not being used, and also conflicting with the newest version of py-evm

Reply:
For context I think it's better to leave a comment about these in the PR or in the commit message.

Reply:
Makes sense, thanks